bentoml/_internal/server/service_app.py

```diff
@@ -149,10 +148,9 @@ async def index_view_func(
             media_type="text/html",
         )
 
+    # pylint: disable=unused-argument
```

Comment:
I think this will disable unused-argument check for everything below the lin in this file/

Reply:
should only be for that function?

Reply:
if unused, just use `_` as the arg name.